app/browser/api/ledgerNotifications.js

```diff
@@ -63,9 +72,12 @@ const init = () => {
 }
 
 const onInterval = (state) => {
+  if (process.env.LEDGER_NOTIFICATION) {
+    console.log(new Date().getTime() / ledgerUtil.milliseconds.second)
```

Comment:
does this need to be logged?

Reply:
This is a wip - sorry

Reply:
This will for testing purposes. It will only run with LEDGER_NOTIFICATION=true npm start so that QA can see the seconds that pass